bin/test_integration_rhel8

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash -ex
+
+cleanup() {
+  echo "Cleaning up..."
+  echo "Cleaning up... logs in cleanup.log"
+  docker-compose rm --stop --force
+}
+
+DEBUG="false"
+if [[ "$1" = "--debug" || "$1" = "-d" ]]; then
+  DEBUG="true"
+  shift
+fi
+
+trap 'echo "ERROR: Test script encountered an error!"; docker-compose logs &> cleanup.log; cleanup' ERR
+trap 'cleanup' EXIT
+cleanup
+
+echo "Building API container..."
+docker-compose build
+
+echo "Generating certificate..."
+docker-compose up openssl
+
+echo "Starting Conjur..."
+docker-compose up -d conjur-appliance
+
+function _random_password() {
+   chars='@'
+  { </dev/urandom LC_ALL=C grep -ao '[A-Za-z0-9]' \
+        | head -n$((RANDOM % 12 + 9))
+    echo ${chars:$((RANDOM % ${#chars})):1}   # Random special char.
+  } \
+    | shuf \
+    | tr -d '\n'
+}
+
+echo "Configuring Conjur..."
+export ADMIN_PASSWORD=$(_random_password)
+export DEBUG=$DEBUG
+export CONJUR_AUTHN_API_KEY=$ADMIN_PASSWORD
+export SERVER_MODE="appliance"
+export ACCOUNT="dev"
+export LOGIN="admin"
+export TEST_HOSTNAME=conjur-appliance
+
+# Docker-compose will pick this file up when starting containers
+# so we will have these variables in the container
+cat <<ENV > .env
+CONJUR_AUTHN_API_KEY=$ADMIN_PASSWORD
+DEBUG=$DEBUG
+SERVER_MODE=$SERVER_MODE
+TEST_HOSTNAME=$TEST_HOSTNAME
```

Comment:
my comment disappeared but I suggested we also have the same logic in the test_integration for the regular test_integration file in the ubuntu flow where we bring over TEST_HOSTNAME